index.test.js

```diff
@@ -0,0 +1,111 @@
+const request = require('supertest');
+const server = require('./index');
+
+describe('index.js', () => {
+
+  describe('GET /games', () => {
+
+    it('should return status code 200', async () => {
+
+      const response = await request(server).get('/games');
+
+      expect(response.status).toBe(200);
+
+    });
+
+    it('should return a list of games', async () => {
```

Comment:
I'd suggest being slightly more explicit here and saying an `array` of games rather than a `list`.